defaults/main.yaml

```
@@ -66,6 +66,7 @@ zookeeper_id: 1
 zookeeper_leader_port: 2888
 zookeeper_election_port: 3888
 zookeeper_servers: "{{ groups['zookeeper-nodes'] }}"
+zookeeper_use_servers_inventory_hostname: false
```

Could you please rearrange the variable name slightly, means it will align better with the existing `zookeeper_servers` variable.

```suggestion
zookeeper_servers_use_inventory_hostname: false
```

---

@ObjectiveTruth , this variable still needs to be updated.